src/assets/css/btn.css

```diff
@@ -0,0 +1,26 @@
+.btn {
```

Comment:
I would use tailwind like all other UI components instead of an hardcoded one, also not sure of those variables will work)